mvrx/src/main/kotlin/com/airbnb/mvrx/TestUtil.kt

```diff
@@ -0,0 +1,38 @@
+package com.airbnb.mvrx
+
+
+/**
+ * This singleton can be used to provide mocked states to viewModels for testing.
+ * State can be directly set or set via arguments. Set the mocker to enabled
+ * to have the viewModels use these mocked states.
+ */
+object MvRxMocker {
+
+    /**
+     * Enable or disable whether the viewModels use mocked states
+     */
+    var enabled: Boolean = false
```

Comment:
Using static variables can cause a lot of headaches for tests. This will not be reset between tests. So if some test turns this on and forgets to turn it off it will stay on for all remaining tests.